modules/src/joke.py

```diff
@@ -1,14 +1,30 @@
 import requests
 from templates.text import TextTemplate
+from random import choice
+import config
 
 def process(input, entities=None):
     output = {}
     try:
+        '''
         r = requests.get('http://tambal.azurewebsites.net/joke/random')
         data = r.json()
         output['input'] = input
         output['output'] = TextTemplate(data['joke']).get_message()
-        output['success'] = True
+        output['success'] = True        
+        '''
+        
+        with open(config.JOKES_SOURCE_FILE,"r") as file:
+            dump = file.read() # read the whole file
+            jokes_list = dump.split("\'''")[2].split("\n")
```

Comment:
ok, consider doing something like this:
rename jokes.txt to jokes.json and add the following data in it:
```
{
	"description": "the comment block for beginners",
	"jokes": [
		"joke #1",
		"joke #2",
		"... and so on..."
	]
}
```